src/elements/content-preview/ContentPreview.js

```diff
@@ -1163,6 +1166,10 @@ class ContentPreview extends React.PureComponent<Props, State> {
                 },
             });
         }
+
+        if (viewer) {
+            viewer.emit('scrolltoannotation', annotation);
```

Comment:
Does this fix the issue where, after creating an annotation, the Preview slightly scrolls the annotation into view?

Reply:
We would need to remove the line in BaseAnnotator that scrolls when the active id is set, which we can do once this change is merged.

Reply:
The overall move to decouple scrolling with the active state should fix that.